applications/cheap_flights/data_manager.py

```diff
@@ -34,26 +34,27 @@ def update_airline_name(airline_id) -> str:
         return airline_id
 
 
-def format_flight_route(flight_routes) -> str:
+def format_flight_route(flight_routes) -> list[str]:
     """ Format the flight route information into a string """
 
     def format_datetime(datetime_str):
-        date = datetime.fromtimestamp(datetime_str)
         """ Format datetime object as string 'dd/mm/yy at 00:00 hours'"""
+        date = datetime.fromtimestamp(datetime_str)
         return date.strftime('%d/%m/%y at %H:%M')
```

Comment:
I'm surprised this works.  The paste you shared says the format of the local_departure is `"local_departure": "2024-02-16T22:40:00.000Z"` but the docstring for `datetime.fromtimestamp` say it takes a float, not a string.  Are you sure this works?

Reply:
yeah, so actually I might have sent you a wrong data,

I pulled form website they have a new version, i might have sent you the old one.

I'll capture the date in a file from my flight app and send on your way.